Makefile

```diff
@@ -16,7 +16,7 @@ $(LOCALBIN):
 
 ENVTEST_K8S_VERSION := 1.28.3
```

Comment:
we should add a make target to copy in the velero crds to simplify that going forward